nifi-nar-bundles/nifi-azure-bundle/nifi-azure-processors/src/main/java/org/apache/nifi/processors/azure/AbstractAzureDataLakeStorageProcessor.java

```diff
@@ -31,6 +31,8 @@
 import com.azure.storage.common.StorageSharedKeyCredential;
 import com.azure.storage.file.datalake.DataLakeServiceClient;
 import com.azure.storage.file.datalake.DataLakeServiceClientBuilder;
+import com.azure.identity.ManagedIdentityCredential;
+import com.azure.identity.ManagedIdentityCredentialBuilder;
```

Comment:
Please use Optimize / Organize Imports (depending on yout IDE).
In the other java files too.
Thanks

Reply:
not sure what you recommend here. Would you help me understand? Do you mean sorting the import based on package names?

Reply:
@sjyang18 Yes, you can do it easily in your IDE. I didn't mean manual sorting.

Reply:
done